services/ipqsservicesample/ui/index.php

```diff
@@ -0,0 +1,257 @@
+          
+          if (/^\s*$/.test(apiKeyTokenValue)) {
+            alert('Please provide a valid API Key Token');
+            inputApiKeyToken.value = '';
+            return;
+          }
+          
+          $.ajax({
+            type: 'PUT',
+            url: apiEndpoints.KEY,
+            contentType: 'application/json',
+            data: JSON.stringify({key: apiKeyTokenValue}),
+          }).done(function (response, status) {
+            textareaApiScoringResult.value = JSON.stringify(response);
+          })
+          
+          apiKeyToken = apiKeyTokenValue;
+          inputApiKeyToken.value = '';
+          inputApiKeyToken.placeholder = 'API Key Token is Set (Click Show to Display It)';
+        }
+        
+        buttonClearApiKeyToken.onclick = function() {
+          $.ajax({
+            type: 'DELETE',
+            url: apiEndpoints.KEY,
+          }).done(function (response, status) {
+            textareaApiScoringResult.value = JSON.stringify(response);
+          })
+          
+          apiKeyToken = undefined;
+          inputApiKeyToken.value = '';
+          inputApiKeyToken.placeholder = 'Enter API Key Token';
+        }
+        
+        buttonShowApiKeyToken.onclick = function() {          
+          $.get(apiEndpoints.KEY, function(response, status) {
+            textareaApiScoringResult.value = JSON.stringify(response);
+          })
+        }
+        
+        buttonExecuteApiScoringEmail.onclick = function() {
+          let email = inputEmail.value;
+          let emailEndpoint = apiEndpoints.EMAIL + `${email}`;
+          
+          $.get(emailEndpoint, function(response, status) {
+            textareaApiScoringResult.value = JSON.stringify(response);
+          })
+        }
+        
+        buttonExecuteApiScoringPhone.onclick = function() {
+          let phone = inputPhone.value;
+          let phoneEndpoint = apiEndpoints.PHONE + `${phone}`;
+          
+          $.get(phoneEndpoint, function(response, status) {
+            textareaApiScoringResult.value = JSON.stringify(response);
+          })
+        }
+        
+        buttonExecuteApiScoringUrl.onclick = function() {
+          let url = inputUrl.value;
+          let urlEndpoint = apiEndpoints.URL + `${url}`;
+          
+          $.get(urlEndpoint, function(response, status) {
+            textareaApiScoringResult.value = JSON.stringify(response);
+          })
+        }
+        
+        buttonExecuteApiScoringIp.onclick = function() {
+          let ip = inputIp.value;
+          let ipEndpoint = apiEndpoints.IP + `${ip}`;
+          
+          $.get(ipEndpoint, function(response, status) {
+            textareaApiScoringResult.value = JSON.stringify(response);
+          })
+        }
+        
+        buttonExecuteApiScoringAddress.onclick = function() {
+          textareaApiScoringResult.value = `Result from Street Address Scoring API: ${inputAddress.value}`;
```

Comment:
Let's either go ahead and implement address scoring interactions, and if not, let's remove it from the UI.